### ISSUE TITLE
video_tag fallback image missing alt attribute

### ISSUE THREAD

**Issue by bigskillet** [Shopify/liquid#1720]
`video_tag` fallback image missing alt attribute and triggering Lighthouse accessibility error:
https://shopify.dev/docs/api/liquid/filters/video_tag

![image](https://github.com/Shopify/liquid/assets/682013/95bb2801-d32d-4612-a801-8ceff515915b)

**Comment by ryanm-bw** [comment]
@Shopify could we please get this one actioned? It should be a simple change and is then consistent with `img_tag`, as well as complying with Lighthouse.

**Comment by PreranaSunbowl** [comment]
Can this be fixed pelase?

**Comment by mcvmel** [comment]
Ran into this as well recently. This may depend on your theme, but after uploading the video in the theme customizer, you can click on the video thumbnail and a black overlay will appear. In that screen there is a space for alt text. This fixed my issue.

**Comment by mj991218** [comment]
plus one

**Comment by mj991218** [comment]
> Thanks man this works but I really wanted to make it automated using product title or any other stuff

Glad it works! For auto-filling alt text with product titles, you can do easily by using GraphQL API, but that’s separate from this issue.
